README.rst

```diff
@@ -4,10 +4,10 @@ baseplate.py
 |Build Status|
 
 It's much easier to manage a bunch of services when they all have the same
-shape: how they're developed, how they interact with the infrastructure, and
-how they interact with eachother. Baseplate is reddit's specification for the
-common shape of our services, and this library, Baseplate.py, is the Python
-implementation of that specification.
+shape: the way they're developed, the way they interact with the infrastructure
+they run on, and the way they interact with each other.  Baseplate is reddit's
+specification for the common shape of our services, and this library,
```

Comment:
Shouldn't Reddit be capitalized?

Reply:
:popcorn:

Reply:
Or maybe not? I guess it's not in the logo. I'm not so sure now. It is a proper noun though. 🤷‍♂️
I only called it out because my last company was very strict about always capitalizing the brand because it might get confused with a fruit otherwise 😛

Reply:
Fixed. re: capitalization... :hear_no_evil: :innocent: :)

Reply:
(brand guidelines aside, we are actually incorporated as `reddit, Inc.` :)